### PR TITLE
.github: add v1.3 to pr template

### PR DESCRIPTION
**Pull request by ran-huang**
Signed-off-by: Ran <huangran@pingcap.com>

### First-time contributors' checklist 

- [ ] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-tidb-operator) that's required for repo owners to accept my contribution.

### Which TiDB Operator version(s) do your changes apply to? (Required)

- [ ] master (the latest development version)
- [ ] v1.2 (TiDB Operator 1.2 versions)
- [ ] v1.1 (TiDB Operator 1.1 versions)
- [ ] v1.0 (TiDB Operator 1.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from: 
- Other reference link(s): 

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Review by TomShawn — APPROVED** [review]

File: .github/pull_request_template.md [hunk 1]
```diff
@@ -23,13 +24,3 @@
 
 - This PR is translated from: <!--Give links here-->
 - Other reference link(s): <!--Give links here-->
-
-### Do your changes match any of the following descriptions?
```
Comment (TomShawn) [inline .github/pull_request_template.md:27]: 为什么要删掉这一部分呢？

Comment (ran-huang) [reply .github/pull_request_template.md:27]: 我感觉这个 repo 很久很久都没用过这部分了😂

**Comment by TomShawn** [comment]
LGTM

**Pull request merged**